modules/dkimVerifier.jsm.js

```diff
@@ -1551,11 +1561,11 @@ var that = {
 				try {
 					from = msgHeaderParser.extractHeaderAddressMailboxes(author);
 				} catch (error) {
-					throw new DKIM_InternalError("From address is ill-formed",	"DKIM_INTERNALERROR_INCORRECT_FROM");
+					throw new DKIM_Error("DKIM_INTERNALERROR_INCORRECT_FROM");
```

Comment:
`DKIM_Error` is not taking a localized string. To avoid confusion that we expect localization to happen here I would probably make `From address is ill-formed` the argument.